<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tl_gestao_estoque</class>
 <widget class="QDialog" name="tl_gestao_estoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>561</width>
     <height>391</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_novo_produto">
    <attribute name="title">
     <string>Novo Produto</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>321</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Código do Produto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_codigo_produto"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Descrição do Produto</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_descricao_produto"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Quantidade em Estoque</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_qntd_estoque"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Fornecedor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_fornecedor"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Valor de Compra</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_valor_compra"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Valor de Venda</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_valor_venda"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btn_novo_produto">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Limpar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_gravar_novo_produto">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Gravar Novo Produto</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_gestao_produtos">
    <attribute name="title">
     <string>Gestão dos Produtos</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>551</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Código do Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_codigo_produto"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_produto"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Quantidade em Estoque</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_qntd"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Valor de Compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_valor_compra"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Valor de Venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_valor_venda"/>
          </item>
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_fornecedor"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tw_ge_produtos"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>551</width>
       <height>97</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Filtrar Por:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QRadioButton" name="rb_ge_codigo">
             <property name="cursor">
              <cursorShape>SizeAllCursor</cursorShape>
             </property>
             <property name="text">
              <string>Código do Produto</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rb_ge_produto">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Descrição do Produto</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Filtrar</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLineEdit" name="txt_ge_filtrar"/>
          </item>
          <item>
           <widget class="QPushButton" name="btn_ge_filtrar">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>FIltrar</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>320</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="btn_ge_gravar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Gravar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ge_excluir">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Excluir</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
